package.json

```diff
@@ -66,56 +66,26 @@
 		"wildemitter": "^1.2.1"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.3",
-		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
-		"@babel/preset-env": "^7.14.4",
-		"@babel/runtime": "^7.14.0",
+		"@nextcloud/babel-config": "^1.0.0-beta.1",
 		"@nextcloud/browserslist-config": "^2.1.0",
-		"@nextcloud/eslint-config": "^1.2.0",
-		"@nextcloud/eslint-plugin": "^1.5.0",
+		"@nextcloud/eslint-config": "^5.1.0",
+		"@nextcloud/stylelint-config": "^1.0.0-beta.0",
+		"@nextcloud/webpack-vue-config": "^4.0.3",
 		"@vue/cli-plugin-unit-jest": "^4.5.13",
 		"@vue/cli-service": "^4.5.13",
 		"@vue/test-utils": "^1.2.0",
-		"babel-eslint": "^10.1.0",
-		"babel-loader": "^8.2.2",
 		"babel-loader-exclude-node-modules-except": "^1.2.1",
-		"babel-plugin-add-module-exports": "^1.0.4",
-		"css-loader": "^5.0.1",
-		"eslint": "^5.16.0",
-		"eslint-config-standard": "^12.0.0",
-		"eslint-import-resolver-webpack": "^0.13.1",
-		"eslint-loader": "^3.0.4",
-		"eslint-plugin-import": "^2.23.4",
-		"eslint-plugin-jest": "^24.3.6",
-		"eslint-plugin-node": "^11.1.0",
-		"eslint-plugin-promise": "^4.3.1",
-		"eslint-plugin-standard": "^4.1.0",
-		"eslint-plugin-vue": "^5.2.3",
 		"jest-localstorage-mock": "^2.4.14",
 		"jest-mock-axios": "^4.4.0",
 		"jest-mock-console": "^1.1.0",
-		"node-sass": "^5.0.0",
 		"regenerator-runtime": "^0.13.7",
-		"sass": "^1.34.1",
-		"sass-loader": "^10.1.1",
-		"style-loader": "^2.0.0",
-		"stylelint": "^13.13.1",
-		"stylelint-config-recommended-scss": "^4.2.0",
-		"stylelint-scss": "^3.19.0",
-		"stylelint-webpack-plugin": "^2.1.1",
-		"url-loader": "^4.1.1",
-		"vue-loader": "^15.9.7",
-		"vue-template-compiler": "^2.6.14",
```

Comment:
why remove this ? the jest test failure is expecting this lib to exist

Reply:
okay, I see, it becomes an implicit dep: https://github.com/nextcloud/webpack-vue-config/blob/master/package.json#L39

not sure why it's not found, probably Jest needs some extras as it doesn't use webpack